lib/utils.js

```diff
@@ -636,7 +642,7 @@ function uniq(a) {
   }
   assert(Array.isArray(a), 'array argument is required');
   var comparableA = a.map(
-    item => item.hasOwnProperty('_bsontype') ? item.toString() : item
+    item => isBsonType(item) ? item.toString() : item
```

Comment:
This change is important - it's fixing compatibility with new version of `bson`.